en/docs/learn/design-api/test-api/test-an-api.md

```diff
@@ -0,0 +1,56 @@
+# Test APIs through the publisher test console
+
+**Testing APIs** is trying out the APIs at the publisher itself and make sure whether the functionalities and behaviours are  met before publishing to the gateway for accessibility of the subscribers.
+
+Only the API developers (creator, publisher) are allowed to test the APIs through the test console. The developers can do the basic functional tests such as the mediation policies, make sure whether the request goes as expected to the back-end
+and the proper responses are received, request/response schema validation and check whether the API resources are defined as expected.
+
+The **Test Console** menu/option in the Publisher portal's API details page can be used to initialize the test at the design phase. Once the developer initiates the test by clicking on the initialize test button, the API is transformed to the prototype(testing)
```

Comment:
The **Test Console** menu item in the Publisher portal's left navigation menu can be used to initialize